### DISCUSSION TITLE
`gambit-scheme` does not link any executables

### DISCUSSION THREAD

**Discussion by jacob-greenfield** [Homebrew/discussions#3587] [Everyday usage] [answered]
After installing `gambit-scheme`, none of the executables appear in my prefix. In fact, trying to link it gives the following output:

```
$ brew unlink gambit-scheme
Unlinking /opt/homebrew/Cellar/gambit-scheme/4.9.3_2... 0 symlinks removed.

$ brew link -n gambit-scheme
Would link:

```

Is this just a fluke on my machine or is the formula broken?

**Comment by SMillerDev** [comment, marked as answer]
Gambit scheme installs their binaries in `$(brew --prefix gambit-scheme)/current/bin` so you'll have to add that to your path.
